go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/vmware-tanzu/carvel-ytt v0.40.0
 	github.com/vmware-tanzu/tanzu-cli/test/e2e/framework v0.0.0-00010101000000-000000000000
 	github.com/vmware-tanzu/tanzu-framework/capabilities/client v0.0.0-20230523145612-1c6fbba34686
-	github.com/vmware-tanzu/tanzu-plugin-runtime v1.2.0-dev.0.20240122215628-021d60b78abe
+	github.com/vmware-tanzu/tanzu-plugin-runtime v1.2.0-dev.0.20240204224748-25dae56d7026
```

Comment:
TODO: Needs to be updated once https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/157 is merged.